### ISSUE TITLE
Mention that config-dist.php file has additional options, at installation stage

### ISSUE THREAD

**Issue by philwareham** [textpattern/textpattern#1531]
### Is your feature request related to a problem?

Not really, just a helpful note.

### What is the feature?

When a user is going through the install process, it might be pertinent to mention on the 'Create config' page that the file `config-dist.php` contains additional setting and features a user might want to use over and above what the create config example does, which is the absolute minimum config requirement.

So, basically, a paragraph under the textarea that states something like (wording can be tweaked):

```
<p>Additional configuration options you may want to use are described in the <code>config-dist.php</code> file that came as part of your installation.</p>
```

There's some nice extra settings in that file that users may never realise exist.

**Comment by Bloke** [comment]
Nice idea.

**Comment by philwareham** [comment]
Cool, are you and @petecooper OK with the wording in the text I suggested, or do you want to improve - since you two are the wordsmiths amongst us?

**Comment by petecooper** [comment]
```
Please refer to <code>textpattern/config-dist.php</code> for optional configuration settings.
```

**Comment by Bloke** [comment]
^ This. With my only reservation being that it night not be `textpattern/` if it's a multi-site install. So we either need to pass in the path (assuming we know it at this stage) or omit it.

**Comment by petecooper** [comment]
```
Please refer to <code>config-dist.php</code> for optional configuration settings.
```

**Comment by petecooper** [comment]
FFS, sorry.

**Comment by petecooper** [comment]
Most people won't need to touch configs, and I'm mindful that some people will see all these options and then panic, open a support thread, and need to be talked down. The route with least friction (most frictionless? ugh) is probably best.

**Comment by philwareham** [comment]
So, no extra text?

**Comment by petecooper** [comment]
Personally I think a note is fine - the UX of an install is to get it done so they can start playing, building etc. Like unwrapping a gift.

If anything, there's more scope to have this more verbosely an article paragraph _after_ the install rather than where it might get lost in an install paragraph.

**Comment by Bloke** [comment]
True. The main benefit to offering it at that moment of creating/uploading config.php is that anyone with non-standard setups can take a look at the examples and perhaps do something about it in their own config.php _before_ logging in and finding nothing works because they require SSL connections, or their server requires a different collation, or something. Maybe they want to enable CSP immediately to save having to do it later, or can find out about the optional UI elements that can be toggled by setting a 'define' in config.php.

It's edge casey, but a simple note that the file presented is the bare bones config and there are a host of other things that can be done (and, btw, here they are if you go look) is beneficial.

**Comment by Bloke** [comment]
Another option is to add it as a comment _inside_ config.php:

```
<?php
$txpcfg['db'] = 'somedb';
$txpcfg['host'] = 'whatever';
...
// For more customisation options, consult config-dist.php.
?>
```

Bonus: no translation string required, as we assume that config.php is English: )

**Comment by petecooper** [comment]
I’m afk at the moment but looping in https://github.com/textpattern/textpattern/issues/1444 for completeness.

-- 
Pete Cooper
pete@pragmatika.net

**Comment by Bloke** [comment]
Why do I not remember writing these things?! I seem to end up repeating myself instead of linking in stuff I've written before. *sigh* Brain.

**Comment by philwareham** [comment]
OK, let's go with the comment in generated config.php then - I like that, and easiest to implement too!

**Comment by philwareham** [comment]
Done. I did also take out the closing tag from the generated PHP, as:

1. It's not needed.
2. If people do start copy/pasting from config-dist.php, it avoids them accidentally pasting after the closing tag.
